cli/flox/doc/manifest.toml.md

```diff
@@ -359,6 +360,13 @@ Semver ::= {
     Setting this value to `true` would allow a package version `4.2.0-pre`
     rather than `4.1.9`.
 
+`cuda-detection`
+:   Whether to detect CUDA libraries and provide them to the environment.
+    The default is `true`.
+    When enabled, Flox will detect if you have an Nvidia device and attempt to
+    locate `libcuda` in well-known paths. Then it will symlink the libraries
+    into `.flox/lib` and add that path to `FLOX_ENV_LIB_DIRS`.
```

Comment:
This is an abridged version of @tomberek's description in https://github.com/flox/flox/issues/1631#issuecomment-2187850369 because I didn't want to make this too wordy or include too much detail about the implementation in case we change it at a later date. If we want to document the ability to manipulate `FLOX_ENV_LIB_DIRS` then we should probably do it elsewhere and with some caution. Suggestions welcome.